specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1568 @@
+            "$ref": "./examples/Alerts_Summary.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.AlertsManagement/alerts/{alertId}/enrichments": {
+      "get": {
+        "operationId": "Alerts_ListEnrichments",
+        "description": "List the enrichments of an alert. It returns a collection of one object named default.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/scope"
+          },
+          {
+            "$ref": "#/parameters/alertId"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the enrichments of the specified alert.",
+            "schema": {
+              "$ref": "#/definitions/alertEnrichmentsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Resolve": {
+            "$ref": "./examples/Alerts_ListEnrichments.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.AlertsManagement/alerts/{alertId}/enrichments/default": {
+      "get": {
+        "operationId": "Alerts_GetEnrichments",
+        "description": "Get the enrichments of an alert.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/scope"
+          },
+          {
+            "$ref": "#/parameters/alertId"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns the enrichments of the specified alert.",
+            "schema": {
+              "$ref": "#/definitions/alertEnrichmentResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Resolve": {
+            "$ref": "./examples/Alerts_GetEnrichments.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "scope": {
+      "name": "scope",
+      "description": "scope here is resourceId for which alert is created.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-skip-url-encoding": true,
+      "x-ms-parameter-location": "client"
+    },
+    "subscriptionId": {
```

Comment:
>subscriptionId [](http://example.com/codeflow?start=5&length=14)

Not used any more? Note, if you put back the subscription scope, you can use the subscriptionId parameter from common types, so you still won't need this one.

Reply:
Looks like you can/should remove subscriptionId parameter, since it's not referenced any more. Not holding ARM signoff for this, but recommended.